Ensure that CMake recurses available paths if the Fortran libs aren't available in the netCDF module

James Carpenter 2021-01-23
--- cmake/jedbrown/FindNetCDF.cmake.orig	2020-04-28 02:19:51.000000000 +0100
+++ cmake/jedbrown/FindNetCDF.cmake	2021-01-25 11:41:47.005657000 +0000
@@ -52,6 +52,7 @@
 
 macro (NetCDF_check_interface lang header libs)
 	find_path (NetCDF_INCLUDES_${lang} NAMES ${header} HINTS "${NetCDF_INCLUDES}" NO_DEFAULT_PATH)
+	find_path (NetCDF_INCLUDES_${lang} NAMES ${header} HINTS "${NetCDF_INCLUDES}")
   
 	find_library (NetCDF_LIBRARIES_${lang} NAMES ${libs} HINTS "${NetCDF_lib_dirs}")
 
